pkg/services/udp_proxy.go

```diff
@@ -19,17 +19,17 @@ type NetcForUDPProxy interface {
 	ListenPacketAndAdvertise(service string, tags map[string]string) (netceptor.PacketConner, error)
 }
 
-type Net struct{}
+type NetUDP struct{}
```

Comment:
I know you didnt create this, but to help with readability, also using the "wrapper" name would be good please